src/backend/index.js

```diff
@@ -123,7 +123,18 @@ function scan () {
 
 function walk (node, fn) {
   if (node.childNodes) {
-    Array.prototype.forEach.call(node.childNodes, function (node) {
+    node.childNodes.forEach(function (node) {
```

Comment:
Hi Evan, happy valentines 😂

Chrome just updated me to 3.0.7.
Console now says:
```
Uncaught TypeError: e.childNodes.forEach is not a function
```

I guess this might be the source? Incredible turnaround time though :ship: